drivers/hmis/app/models/hmis/hud/custom_assessment.rb

```diff
@@ -150,6 +150,8 @@ def title
     title
   end
 
+  alias contact_date assessment_date
```

Comment:
I'm not sure if this is worth adding aliases to the HMIS models, since we're only using it in a couple places. This could cause issues later if there were to be a "contact date" added to any hud model (top of mind due to recent discussions around a new date field on Service).

Could this logic be encapsulated in a utility function instead? e.g. `contact_date_for_entity` on Enrollment